seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -714,31 +706,15 @@
       "title": "class getters and setters",
       "description": [
         "You can obtain values from an object, and set a value of a property within an object.",
-        "These are classically called getters and setters.",
+        "These are classically called <dfn>getters and setters</dfn>.",
         "Getter functions are meant to simply return (get) the value of an object's private variable to the user without the user directly accessing the private variable.",
         "Setter functions are meant to modify (set) the value of an object's private variable based on the value passed into the setter function. This change could involve calculations, or even overwriting the previous value completely.",
-        "<code>class Book {</code>",
-        "<code>  constructor(author) {</code>",
-        "<code>    this._author = author;</code>",
-        "<code>  }</code>",
-        "<code>  // getter</code>",
-        "<code>  get writer(){</code>",
-        "<code>    return this._author;</code>",
-        "<code>  }</code>",
-        "<code>  // setter</code>",
-        "<code>  set writer(updatedAuthor){</code>",
-        "<code>    this._author = updatedAuthor;</code>",
-        "<code>  }</code>",
-        "<code>}</code>",
-        "<code>const lol = new Book('anonymous');</code>",
-        "<code>console.log(lol.writer);</code>",
-        "<code>lol.writer = 'wut';</code>",
-        "<code>console.log(lol.writer);</code>",
+        "<blockquote>class Book {<br>  constructor(author) {<br>    this._author = author;<br>  }<br>  // getter<br>  get writer(){<br>    return this._author;<br>  }<br>  // setter<br>  set writer(updatedAuthor){<br>    this._author = updatedAuthor;<br>  }<br>}<br>const lol = new Book('anonymous');<br>console.log(lol.writer);<br>lol.writer = 'wut';<br>console.log(lol.writer);</blockquote>",
         "Notice the syntax we are using to invoke the getter and setter - as if they are not even functions.",
         "Getters and setters are important, because they hide internal implementation details.",
-        "Instructions",
-        "Use class keyword to create a Thermostat class. The constructor accepts Farenheit temperature.",
-        "Now create getter and setter in the class, to obtain the temperature in Celsius scale.",
+        "<hr>",
+        "Use <code>class</code> keyword to create a Thermostat class. The constructor accepts Farenheit temperature.",
+        "Now create <dfn>getter and setter</dfn> in the class, to obtain the temperature in Celsius scale.",
```

Comment:
[Same change suggestion](https://github.com/freeCodeCamp/freeCodeCamp/pull/13559/files#r103077540) should be made here as well.

```diff
- <dfn>getter and setter</dfn>
+ <dfn>getter</dfn> and <dfn>setter</dfn>
```